src/Search/FluentSearchVariant.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace TractorCow\Fluent\Search;
+
+use TractorCow\Fluent\Extension\FluentExtension;
+use TractorCow\Fluent\State\FluentState;
+use TractorCow\Fluent\Model\Locale;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\FullTextSearch\Search\Variants\SearchVariant;
+use SilverStripe\FullTextSearch\Search\SearchIntrospection;
+use SilverStripe\FullTextSearch\Search\Queries\SearchQuery;
+
+if (!class_exists(SearchVariant::class)) {
+    return;
+}
+
+class FluentSearchVariant extends SearchVariant
```

Comment:
Maybe adding a doc block might be useful to explain what it does at a high level?

Reply:
Sounds ideal.

Reply:
Done :)